kotsadm/pkg/apiserver/server.go

```diff
@@ -87,6 +94,9 @@ func Start() {
 	// This the handler for license API and should be called by the application only.
 	r.Path("/license/v1/license").Methods("GET").HandlerFunc(handlers.GetPlatformLicenseCompatibility)
 
+	// This handler proxies all requests with prefix /dex to the Dex identity proxy
+	r.PathPrefix("/dex/").HandlerFunc(upstreamDex)
```

Comment:
should this be `/dex` instead of `/dex/`?

Reply:
I dont know. we can change it if it causes an issue